src/BrazeKit-dev.js

```diff
@@ -394,6 +396,14 @@ var constructor = function() {
                     options.baseUrl = customUrl;
                 }
             }
+            
+            if (mpCustomFlags && mpCustomFlags[moduleId]) {
```

Comment:
Annoyingly, the key of the object can't be a number.  It's a string, so it'll have to be:

```suggestion
            if (mpCustomFlags && mpCustomFlags[moduleId.toString()]) {
```